index.html

```diff
@@ -3,10 +3,11 @@
 <head>
   <meta charset="utf-8">
   <link rel="stylesheet" href="styles.css">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0"
   </head>
 <body>
   <nav class="navbar">
-    <a href="#title"><img class="header-logo" src="https://www.guiaviagensbrasil.com/imagens/Imagem%20de%20surfistas%20no%20mar%20em%20Fernando%20de%20Noronha-Pernambuco.jpg"/></a>
+    <a href="#title"><img class="header-logo" src="https://www.guiaviagensbrasil.com/imagens/Imagem%20de%20surfistas%20no%20mar%20em%20Fernando%20de%20Noronha-Pernambuco.jpg"/></a
```

Comment:
1 - Please use the correct syntax for <a> tag.

2 - Why did you not consider a element "title" in the Element “head”? This Element  is missing a required instance of this child element.

3 - An “img” element must have an “alt” attribute, except under certain conditions. Please include this change in order to have the right syntax.